README.md

```diff
@@ -1,39 +1,84 @@
 # LochNessBuilder
 
-A C# Builder library to automate and commonise creation of objects for tests.
-You can define a basic builder that wires up default values, which all tests can then use as-is, or add further customisations on top of that basic builder for specific tests.
+LochNessBuilder is a C# Test Object Builder library to automate and commonise creation of objects for tests.
+The intended usage is NOT that you create a new builder for each test, or even for each test class. Rather, that you'll define one or two builders for each major object in your domain, which set up a set of sensible default values for those objects, and set up relationships with all the other major objects that it a given test object linked to.
+Then in the individual tests or test classes, you use the default builder and apply further configurations **only** to those additional properties that you *actively* care about in those specific tests.
 
-Note that v3.0 has made a lot of superficial changes to the API. See below for details
+Note that v4.0 has made a lot of superficial changes to the API. See below for details.
 
 ## Migration Guide
 
 ### **Do not use v2.0.0**
 
 This version was released inadvertantly and has been deprecated and de-listed from nuget.org
+Versions `2.1` and `4.0` are available, and the latter should be used if possible.
 
-Versions `2.1` and `3.0-alpha` are available
-
-* If you wanted "`v1.x` but on .NET Standard", then use v2.1.
-* If you really wanted "the version that was released as v2.0.0" then use v3.0-alpha, though this is not release-ready and should only be used if you've already integrated against the inadvertant v2.0.0 release.
+* If you wanted "`v1.x` but on .NET Standard", then use `v2.1`.
+* If you wanted a version with the updated API, then use `v4.0`, and see migration notes below.
```

Comment:
Uhhh ... I've wildly lost track of what's happened with .NET in the last year or two.
As far as I *know* ... everything? (or at least ... everything since before .NET Framework 4.? )
Certainly if that's *not* the case, we should try to *make* it the case.

I'll do some brief testing.

Reply:
Have clarified this in the comment. Actual testing is still pending.

Reply:
Tracked in Slack post